doc/classes/EditorImportPlugin.xml

```diff
@@ -35,15 +35,15 @@
 		func get_import_options(i):
 		    return [{"name": "my_option", "default_value": false}]
 
-		func load(src, dst, opts, r_platform_variants, r_gen_files):
+		func import(source_file, save_path, options, r_platform_variants, r_gen_files):
 		    var file = File.new()
-		    if file.open(src, File.READ) != OK:
+		    if file.open(source_file, File.READ) != OK:
 		        return FAILED
 
 		    var mesh = Mesh.new()
 
-		    var save = dst + "." + get_save_extension()
-		    ResourceSaver.save(file, mesh)
+		    var filename = save_path + "." + get_save_extension()
+		    ResourceSaver.save(filename, mesh)
```

Comment:
It saves an empty Mesh regardless of `source_file` no?

Reply:
Yes, but earlier sample kinda mislead users into overwriting the source file since it saves to `file` instead of `save`.

Reply:
Reposting here (and bumping :)) as the email answer was messy:

> Sure, but it would be nice to make the new sample non-useless :P

Reply:
Personally I think it's good enough for built-in docs, but I'm fine with real example too, maybe not with Mesh because it would be complicating to me and probably the readers. This [tutorial](http://docs.godotengine.org/en/3.0/tutorials/plugins/editor/import_plugins.html) already has a working sample for those interested to learn more. I linked the tutorial in the latest commit.

Reply:
I don't think it would be complex, this example was just to add support for custom extensions AFAIK, so it's probably possible to load the mesh with `ResourceLoader` directly before saving it again with the new extension.

If not, then it would be better to have a completely different example that actually does something, even trivial.

Reply:
I can't seem to use `ResourceLoader` to load custom extension even with type hint. I'll try something else later.

Reply:
Since there is an `EditorImportPlugin` tutorial I agree with @Noshyaar that this example is sufficient enough to get the usage across

Reply:
Well I'll merge as it's better than it was, but I still think it should be made more useful.